clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1085,12 +1085,19 @@ static target getAccessTarget(QualType FieldTy,
       AccTy->getTemplateArgs()[3].getAsIntegral().getExtValue());
 }
 
-// FIXME: Free functions must have void return type, be declared at file scope,
```

Comment:
You've lost the part of the comment about requiring a `void` return type.

Reply:
Fixed.